pkgs/applications/networking/instant-messengers/discord/linux.nix

```diff
@@ -170,7 +170,7 @@ stdenv.mkDerivation rec {
       libdbusmenu
       wayland
     ]
-    ++ lib.optional withTTS speechd-minimal
+    ++ lib.optionals withTTS [ speechd-minimal ]
```

Comment:
Is there a reason why this was changed?
Makes more sense, so I don't think it's a problem, just seems a bit random.